src/apis/sourceimages.js

```diff
@@ -181,7 +181,7 @@ export default (state) => {
         const formData = {}
         if (metadata !== null) {
           Object.keys(metadata).forEach(function (o) {
-            formData[o + '[0]'] = metadata[o]
+            formData[o + '[0]'] = JSON.stringify(metadata[o])
```

Comment:
But that's a BC break now? 
I have code out there, which does exactly what Jan did, stringify in the user code, which would now break on an update. Shouldn't we catch this and only stringify, when it's an object?

Reply:
fixed, nice catch ;)